### PR TITLE
Update for different panel states

### PR DESCRIPTION
**Pull request by cassidyjames**
Not sure what happened with https://github.com/pop-os/gtk-theme/issues/13, but this should be proper panel styling. Based on https://github.com/elementary/stylesheet/blob/199e333e09eef133c7f82b7cc38c75f8436a4031/elementary/gtk-3.0/apps.css#L260.

### CODE REVIEW

**Comment by cassidyjames** [comment]
See it in action at https://youtu.be/ajMArMyjHM4

File: src/gtk3/common/scss/apps/_pantheon-panel.scss [hunk 1]
```diff
@@ -1,12 +1,37 @@
...
-  font-weight: normal;  
+}
+
+.panel.translucent {
+  background-color: rgba($warm_grey_900, 0.5);
+
+}
+
+.panel.color-light.translucent {
+  background-color: rgba(#fff, 0.85);
```
Comment (isantop) [inline src/gtk3/common/scss/apps/_pantheon-panel.scss]:
```
24:21  warning  Only variables or functions must be passed to rgba, literals are restricted  no-color-literals
```

File: src/gtk3/common/scss/apps/_pantheon-panel.scss [hunk 2]
```diff
@@ -1,12 +1,37 @@
...
+}
+
+.composited-indicator > revealer label,
+.composited-indicator > revealer image,
+.composited-indicator > revealer spinner {
+  color: #fff;
```
Comment (isantop) [inline src/gtk3/common/scss/apps/_pantheon-panel.scss]:
```
30:10  warning  Color literals such as '#fff' should only be used in variable declarations   no-color-literals
```

File: src/gtk3/common/scss/apps/_pantheon-panel.scss [hunk 3]
```diff
@@ -1,12 +1,37 @@
...
- */
+/*
+* _pantheon-panel.scss - App overrides for Pantheon Panel
+*
+* File credits: Ian Santopietro <isantop@gmail.com>
+*/
 
 .panel {
+  background-color: transparent;
+  transition: all 100ms ease-in-out;
```
Comment (isantop) [inline src/gtk3/common/scss/apps/_pantheon-panel.scss]:
```
10:3   error    Expected `transition`, found `background-color`                              property-sort-order
11:3   error    Expected `background-color`, found `transition`                              property-sort-order
```

File: src/gtk3/common/scss/apps/_pantheon-panel.scss [hunk 4]
```diff
@@ -1,12 +1,37 @@
...
+.composited-indicator > revealer label,
+.composited-indicator > revealer image,
+.composited-indicator > revealer spinner {
+  color: #fff;
+}
+
+.panel.color-light .composited-indicator > revealer label,
+.panel.color-light .composited-indicator > revealer image,
+.panel.color-light .composited-indicator > revealer spinner {
+  color: rgba(#000, 0.65);
```
Comment (isantop) [inline src/gtk3/common/scss/apps/_pantheon-panel.scss]:
```
36:10  warning  Only variables or functions must be passed to rgba, literals are restricted  no-color-literals
```

**Comment by isantop** [comment]
It looks like there's an additional warning from the linter about _menus.scss. That's unrelated to this change and can be wrt this PR.

**Comment by isantop** [comment]
Lint fixes should be fixed as of d4e3b7b

**Comment by isantop** [comment]
I'm going to close this for now, since it looks somewhat abandoned. Feel free to update the branch and reopen or submit a new PR.

**Pull request closed without merging**